pypeclub/OpenPype
# Maya: Add additional playblast options to review Extractor.

## Brief description
This adds in additional playblast options to the project settings review extractor settings and resolves #3383.

## Description
Propagates flags as keys from `capture.py` into `schema_maya_capture.json` for it to be passed during extraction of `review` instance.


## Testing notes:
1. Change project settings values of the review extractor.
2. Create `review` instance as per OpenPype workflow. (Can be found here: https://openpype.io/docs/artist_hosts_maya#reviews)
3. Publishing should generate playblast with correct settings applied.
